pkg/log/status.go

```diff
@@ -265,6 +265,34 @@ func Swarningf(format string, a ...interface{}) string {
 	return fmt.Sprintf(" %s%s%s", yellow(getWarningString()), suffixSpacing, fmt.Sprintf(format, a...))
 }
 
+// Title Prints the logo as well as the first line being BLUE (indicator of the command information)
+// the second and third lines are optional and provide information with regards to what is being ran
+func Title(firstLine, secondLine, thirdLine string) {
+	// Output information regarding the odo deploy command before running
```

Comment:
Is this specific to odo deploy, or do we intend on using it with all the commands?

Reply:
@valaparthvi has a good point here. That comment is misleading as this is used in `odo dev` as well.